mpu6050_scripts/get_temperature.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#Init
+sudo i2cset -y 0 0x68 0x6B 0
+
+while true
+do
+
+#Read data
+UB=$(sudo i2cget -y 0 0x68 0x41)
+LB=$(sudo i2cget -y 0 0x68 0x42)
```

Comment:
Is it possible to read high and low part of data in single command using i2cget option '-w' ?

Reply:
Yes, sudo i2cget -y 0 0x68 0x41 w works. I didn't know about this. Thanks a lot

Reply:
Like this: UB=$(sudo i2cget -y 0 0x68 0x41) && LB=$(sudo i2cget -y 0 0x68 0x42) && UBLB=$(sudo i2cget -y 0 0x68 0x41 w) && echo $UB $LB $UBLB

Reply:
I suppose command 'sudo i2cget -y 0 0x68 0x41 w' will give You UBLB as result. Is it correct?